packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -489,14 +489,14 @@ export class RichText extends Component {
 
 		if ( typeof pastedContent === 'string' ) {
 			const recordToInsert = create( { html: pastedContent } );
-			const insertedContent = insert( currentRecord, recordToInsert );
-			const newContent = this.valueToFormat( insertedContent );
+			const resultingRecord = insert( currentRecord, recordToInsert );
+			const resultingContent = this.valueToFormat( resultingRecord );
 
 			this.lastEventCount = undefined;
-			this.value = newContent;
+			this.value = resultingContent;
 
 			// explicitly set selection after inline paste
-			this.forceSelectionUpdate( insertedContent.start, insertedContent.end );
+			this.onSelectionChange( resultingRecord.start, resultingRecord.end );
```

Comment:
@Tug can you confirm whether this is the right approach to update the selection after the inline paste event?

Reply:
This looks good to me 👍 
It seems we don't need `forceSelectionUpdate` anymore, right?

Reply:
Good catch! I pushed a commit to remove that function.